pkg/local_object_storage/metabase/exists.go

```diff
@@ -112,7 +112,7 @@ func objectStatus(tx *bbolt.Tx, addr oid.Address, currEpoch uint64) uint8 {
 	expirationBucket := tx.Bucket(attributeBucketName(addr.Container(), objectV2.SysAttributeExpEpoch))
 	if expirationBucket != nil {
 		// bucket that contains objects that expire in the current epoch
-		currEpochBkt := expirationBucket.Bucket([]byte(strconv.FormatUint(currEpoch, 10)))
+		currEpochBkt := expirationBucket.Bucket([]byte(strconv.FormatUint(currEpoch+1, 10)))
```

Comment:
Why do we need this change?

Reply:
changed that line (var naming and epoch decrementing) while was fixing tests